.github/ISSUE_TEMPLATE/release-template-new.md

```
@@ -0,0 +1,69 @@
+**3. Communicate**
+- [ ] Inform Parse.ly support of the upcoming release.
+
+The following additional tasks might be needed depending on the release and its impact:
+- [ ] Write any needed internal documentation.
+- [ ] Write an internal P2 post about the release (to be posted immediately so folks are aware of the release ahead of time).
+- [ ] Write a WordPress VIP Lobby post about the release (to be posted immediately to preannounce next week's VIP release - don't forget to get someone to proofread!).
+- [ ] Prepare any public documentation (to be posted after the WordPress.org release).
+
+## Release process
+
+**1. Update version numbers and changelog**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)